examples/vnet/doca_gw_dpdk.c

```diff
@@ -649,6 +652,202 @@ static int doca_gw_dpdk_build_action(struct doca_gw_pipeline_cfg *cfg,
 	return ret;
 }
 
+static void doca_gw_dpdk_build_end_action(struct doca_gw_pipe_dpdk_flow *pipe)
+{
+	struct rte_flow_action *action = &pipe->actions[pipe->nb_actions++];
+	action->type = RTE_FLOW_ACTION_TYPE_END;
+}
+
+static int 
+doca_gw_dpdk_build_rss_action(struct doca_dpdk_action_entry *entry,
+									struct doca_fwd_table_cfg *fwd_cfg)
+{
+	int qidx;
+	struct rte_flow_action *action = entry->action;
+	struct doca_dpdk_action_rss_data *rss = &entry->action_data.rss;
+
+	rss->conf.queue_num = fwd_cfg->rss.num_queues;
+	for (qidx = 0; qidx < fwd_cfg->rss.num_queues; qidx++)
+		rss->queue[qidx] = fwd_cfg->rss.queues[qidx];
+	rss->conf.func = RTE_ETH_HASH_FUNCTION_DEFAULT;
+	rss->conf.types = ETH_RSS_IP;
```

Comment:
I think you can leave it as 0. if you do ip isn't doing the rss just on ips without TCP/UDP ports?